dataeng/resources/snowflake-collect-queue-depth.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+set -ex
+
+# Setup
+cd $WORKSPACE/analytics-tools/snowflake
+make requirements
+
+python collect_queue_depth.py \
+    --key_path $KEY_PATH \
+    --passphrase_path $PASSPHRASE_PATH \
+    --automation_user $USER \
+    --account $ACCOUNT
```

Comment:
I'm noticing all your files don't end with newlines...it only matters to me inasmuch as github adds a red dash on the last line every time.  I know IDEs can be a bit finnicky about this, maybe you can find a setting to automatically add a newline?